src/jobs/taskcluster_graph.js

```diff
@@ -113,104 +93,54 @@ export default class TaskclusterGraphJob extends Base {
       host: repositoryUrlParts.host
     };
 
-    let errorGraphUrl =
-      mustache.render(this.config.try.errorTaskUrl, urlVariables);
-
-    // Try fetching from .taskgraph.yml, the preferred location, falling back
-    // to the project URL The latter is supported only for branches to which
-    // .taskcluster.yml (and the task-graph generation support in taskcluster/
-    // to which it points) has not yet been merged.
-    try {
-      let graphUrl = projectConfig.tcYamlUrl(this.config.try, urlVariables);
-      console.log(`Fetching '.taskcluster.yml' url ${graphUrl} for '${repo.alias}' push id ${push.id}`);
-      let graphText = await fetchGraph(graphUrl);
-      templateVariables.source = graphUrl;
-      // Assume .taskcluster.yml has been fetched successfully
-      let queue = new taskcluster.Queue({
-        credentials: this.config.taskcluster.credentials,
-        authorizedScopes: scopes
-      });
-
-      return await this.scheduleTaskGroup(queue,
-                                          repo.alias,
-                                          graphText,
-                                          templateVariables,
-                                          scopes,
-                                          errorGraphUrl);
-    } catch (err) {
-      console.log(`Error fetching .taskcluster.yml at ${graphUrl} for ${repo.alias}. ${err.stack}`);
-      let graphUrl = projectConfig.url(this.config.try, repo.alias, urlVariables);
-      console.log(`Fetching url ${graphUrl} for '${repo.alias}' push id ${push.id}`);
-      let graphText = await fetchGraph(graphUrl);
-      templateVariables.source = graphUrl;
-
-      let scheduler = new taskcluster.Scheduler({
-        credentials: this.config.taskcluster.credentials,
-        // Include scopes for creating and extending task graphs.
-        authorizedScopes: scopes.concat(['scheduler:create-task-graph', 'scheduler:extend-task-graph:*'])
-      });
+    let graphUrl = projectConfig.tcYamlUrl(this.config.try, urlVariables);
+    console.log(`Fetching '.taskcluster.yml' url ${graphUrl} for '${repo.alias}' push id ${push.id}`);
+    let graphText = await this.fetchGraph(graphUrl);
+    templateVariables.source = graphUrl;
+    // Assume .taskcluster.yml has been fetched successfully
```

Comment:
Dunno, that was a re-indentation.  I don't think we do assume -- fetchGraph would have raised an exception.